src/prefect/server/api/flow_runs.py

```diff
@@ -364,6 +366,21 @@ async def resume_flow_run(
         if keyset:
             run_input = run_input or {}
 
+            if experiment_enabled("enhanced_deployment_parameters"):
```

Comment:
Not sure if we want a different experiment or not. I think it's probably overkill to have more but just want to align with w/e @pleek91 needs

Reply:
Yeah, I debated it and decided to go this route, but happy to add another if we think it's needed.

Reply:
I assumed we would use the same flag 👍